assets/js/modules/adsense/index.js

```diff
@@ -44,7 +44,7 @@ import AdSenseIcon from '../../../svg/adsense.svg';
 import { STORE_NAME } from './datastore/constants';
 import { ERROR_CODE_ADBLOCKER_ACTIVE, CONTEXT_MODULE_ADSENSE, AREA_MODULE_ADSENSE_MAIN } from './constants';
 import { WIDGET_AREA_STYLES } from '../../googlesitekit/widgets/datastore/constants';
-import { registerStore as registerDataStore } from './datastore';
+import { registerStore as registerDatastore } from './datastore';
```

Comment:
@tofumatt following on from your other comment, this was the only camelCase usage I found in the codebase

Reply:
Nice, thanks!